client/modules/IDE/components/FileNode.jsx

```diff
@@ -105,6 +105,9 @@ export class FileNode extends React.Component {
       'sidebar__file-item--editing': this.state.isEditingName,
       'sidebar__file-item--closed': this.props.isFolderClosed
     });
+
+    let isFocused = false;
```

Comment:
rather than this being a local variable with `render`, this should be stored within the component state. the functions that are called from `onBlur` could be moved outside of render too, methods of `FileNode`.

Reply:
Thank you for the feedback! I'll fix this soon.

Reply:
Fixed it.